libs/ui/select/helm/src/lib/hlm-select-trigger.component.ts

```diff
@@ -46,11 +47,9 @@ type SelectTriggerVariants = VariantProps<typeof selectTriggerVariants>;
 	`,
 })
 export class HlmSelectTriggerComponent {
-	@ViewChild('button', { static: true })
-	public buttonEl!: ElementRef;
+	public buttonEl = viewChild<ElementRef>('button');
```

Comment:
should this one be readonly?

Reply:
this one is not needed and can be removed.